### ISSUE TITLE
Issue compiling examples

### ISSUE THREAD

**Issue by Hyperthetical** [alex-mcdaniel/RX-DMFIT#5]
Hi,

When I run 
`make example1`
or
`make example2` 

I get the following error:

`
pIC.cpp: In static member function ‘static double Target::SL_profile(double)’:
pIC.cpp:67:9: error: ‘r_b_cm’ was not declared in this scope
   67 |         r_b_cm = r_b*kpc2cm;
      |         ^~~~~~
pIC.cpp:68:9: error: ‘r_d_cm’ was not declared in this scope
   68 |         r_d_cm = r_d*kpc2cm;
      |         ^~~~~~
make: *** [makefile:20: example1] Error 1
`

Any suggestion as to what is happening?

**Comment by alex-mcdaniel** [comment]
Yes this looks like its caused by an error in the variable declaration in lines 67 and 68 of pIC.cpp. 
The lines :

```
r_b_cm = r_b*kpc2cm;
r_d_cm = r_d*kpc2cm;

```
should instead read:

```
double r_b_cm = r_b*kpc2cm;
double r_d_cm = r_d*kpc2cm;
```
This was a recent modification and has now been changed in the master code. Give it a try with this fixed and if there are any other issues let me know.
